yarn.lock

```diff
@@ -1296,752 +1296,804 @@
-    "@swc/helpers" "^0.4.14"
-
-"@fluentui/react-dialog@^9.5.15":
-  version "9.5.15"
-  resolved "https://registry.yarnpkg.com/@fluentui/react-dialog/-/react-dialog-9.5.15.tgz#6a9504d0203efddf04b314f2f2ae0f6aace22c7d"
-  integrity sha512-0Mn3YT5i1evNMftL23x3h20/OnPZwQXEIL55rYj8H4Ja5RKrsP69EU/HJ+sNnTv2LHhp1SxzYW2qLpx3jmMVLg==
-  dependencies:
-    "@fluentui/keyboard-keys" "^9.0.3"
-    "@fluentui/react-aria" "^9.3.26"
-    "@fluentui/react-context-selector" "^9.1.26"
-    "@fluentui/react-icons" "^2.0.203"
-    "@fluentui/react-jsx-runtime" "9.0.0-alpha.10"
-    "@fluentui/react-portal" "^9.3.1"
-    "@fluentui/react-shared-contexts" "^9.6.0"
-    "@fluentui/react-tabster" "^9.10.0"
-    "@fluentui/react-theme" "^9.1.9"
-    "@fluentui/react-utilities" "^9.10.1"
-    "@griffel/react" "^1.5.7"
-    "@swc/helpers" "^0.4.14"
-
-"@fluentui/react-divider@^9.2.19":
-  version "9.2.19"
-  resolved "https://registry.yarnpkg.com/@fluentui/react-divider/-/react-divider-9.2.19.tgz#00e72d64972cff88c9e494be3e047e9bbe12ad33"
-  integrity sha512-H7GJby0+FuCUbUE9ydMenZMmxfGMMdmzZJSAWXIRTNJB0ds/29/Na/pNX6gmcYOsaO6IrPwFD3V79MB4tVoiKQ==
-  dependencies:
-    "@fluentui/react-jsx-runtime" "9.0.0-alpha.10"
-    "@fluentui/react-shared-contexts" "^9.6.0"
-    "@fluentui/react-theme" "^9.1.9"
-    "@fluentui/react-utilities" "^9.10.1"
-    "@griffel/react" "^1.5.7"
-    "@swc/helpers" "^0.4.14"
-
-"@fluentui/react-drawer@9.0.0-beta.7":
-  version "9.0.0-beta.7"
-  resolved "https://registry.yarnpkg.com/@fluentui/react-drawer/-/react-drawer-9.0.0-beta.7.tgz#e849e241f2bb8c361c649adf36df51d00def54cd"
-  integrity sha512-RlJoj/9TTqHYdmSXvYUo0C2mh+F66CzLhorBLrfBD35Sg874D3oWLu+bi0SVfynqRRMlKDt7m/a4k3EsgZ6ZtA==
-  dependencies:
-    "@fluentui/react-dialog" "^9.5.15"
-    "@fluentui/react-jsx-runtime" "9.0.0-alpha.10"
-    "@fluentui/react-shared-contexts" "^9.6.0"
-    "@fluentui/react-theme" "^9.1.9"
-    "@fluentui/react-utilities" "^9.10.1"
-    "@griffel/react" "^1.5.7"
-    "@swc/helpers" "^0.4.14"
-
-"@fluentui/react-field@^9.1.10":
-  version "9.1.10"
-  resolved "https://registry.yarnpkg.com/@fluentui/react-field/-/react-field-9.1.10.tgz#b36d7505d5822a7862567e351b7ef1374c87b943"
-  integrity sha512-1Zz2EJs1O6L3oaTWwS/OC4PWpiEO9/YzaUGA35CtSHCVjACTn2fUQY4VRvK40QZhc22n1NS1I6LqhW+Ju2sPew==
-  dependencies:
-    "@fluentui/react-context-selector" "^9.1.26"
-    "@fluentui/react-icons" "^2.0.203"
-    "@fluentui/react-jsx-runtime" "9.0.0-alpha.10"
-    "@fluentui/react-label" "^9.1.19"
-    "@fluentui/react-theme" "^9.1.9"
-    "@fluentui/react-utilities" "^9.10.1"
-    "@griffel/react" "^1.5.7"
-    "@swc/helpers" "^0.4.14"
-
-"@fluentui/react-icons@^2.0.203":
-  version "2.0.204"
-  resolved "https://registry.yarnpkg.com/@fluentui/react-icons/-/react-icons-2.0.204.tgz#cc93486e1cbea5f0df2cd6b1648d47fe6eb06557"
-  integrity sha512-zl1T5CorbJ3cLeM3BfXxT1Bdm1f56gTXagrJG+f+b3gs+bI5qeNmCkZv5yq0DVdWx5KAqYoeX57wkn7GM5GyTA==
+"@fluentui/keyboard-keys@^9.0.6":
```

Comment:
can you please run

`npx yarn-deduplicate`

in your repo root and check in the changes to yarn.lock? I haven't had the time to do some automation checks on that yet unfortunately

Reply:
resolved, thanks